chia/cmds/keys_funcs.py

```diff
@@ -285,9 +285,9 @@ def sign(message: str, private_key: PrivateKey, hd_path: str, as_bytes: bool, js
 
 def verify(message: str, public_key: str, signature: str, as_bytes: bool) -> None:
     data = bytes.fromhex(message) if as_bytes else bytes(message, "utf-8")
-    public_key = G1Element.from_bytes(bytes.fromhex(public_key))
-    signature = G2Element.from_bytes(bytes.fromhex(signature))
-    print(AugSchemeMPL.verify(public_key, data, signature))
+    pk: G1Element = G1Element.from_bytes(bytes.fromhex(public_key))
+    sig: G2Element = G2Element.from_bytes(bytes.fromhex(signature))
+    print(AugSchemeMPL.verify(pk, data, sig))
```

Comment:
the variables `public_key` and `signature` already have types, mypy doesn't allow shadowing the previous variables with new ones of different types

Reply:
Do the `.from_bytes()` methods not hint the proper return type?  The `: G1Element` doesn't seem needed for clarity etc.

Reply:
good point. They are not necessary, I added them to explain to myself how these are new variables with different types